index.js

```diff
@@ -43,6 +43,74 @@ exports.Store = Store;
 exports.Cookie = Cookie;
 exports.Session = Session;
 exports.MemoryStore = MemoryStore;
+exports.sessionId = {
+  /**
+   * Set sessionId to cookie.
+   *
+   * @private
+   */
+  set: function (res, name, sessionId, secret, options) {
```

Comment:
Just reference the old `setcookie` function instead of copy and pasting it up here (where it ends up with another level of indentation).